test/aws-xray-sdk/tc_plugin.rb

```diff
@@ -80,4 +82,53 @@ def test_ec2_metadata_fail
     assert expected, XRay::Plugins::EC2.aws
     WebMock.reset!
   end
+
+  # ECS Plugin
+  def test_ecs_metadata_successful
+    dummy_metadata_uri = 'http://169.254.170.2/v4/a_random_id'
+    dummy_json = {
+      "ContainerARN"=>"arn:aws:ecs:eu-central-1:an_id:container/a_cluster/a_cluster_id/a_task_id",
+      "LogOptions"=>{"awslogs-group"=>"/ecs/a_service_name", "awslogs-region"=>"eu-central-1", "awslogs-stream"=>"ecs/a_service_name/a_task_id"},
+    }
+
+    ENV[XRay::Plugins::ECS::METADATA_ENV_KEY] = dummy_metadata_uri
+    stub_request(:get, dummy_metadata_uri)
+      .to_return(status: 200, body: dummy_json.to_json, headers: {})
+
+    expected = {
+      ecs: {
+        container: Socket.gethostname,
+        container_arn: 'arn:aws:ecs:eu-central-1:an_id:container/a_cluster/a_cluster_id/a_task_id',
+      },
+      cloudwatch_logs: {:log_group=>"/ecs/a_service_name", :log_region=>"eu-central-1", :arn=>"arn:aws:ecs:eu-central-1:an_id:container/a_cluster/a_cluster_id/a_task_id"}
```

Comment:
This is the key change here, with this information present in the trace, the logs should be linked